CONTRIBUTING.md

```diff
@@ -59,8 +59,10 @@ project through Rosetta][63].
 
 ### Windows
 
-Running the project natively on Windows is not currently possible due to [dependency issues][61].
-Instead running on [Ubuntu](#ubuntu-linux) via [Windows Subsystem for Linux][60] is recommended.
+The project can be run from any Windows command line environment provided that Node.js
+and npm are installed. ([Dependency issues][61] which prevented Windows compatibility
+have now been resolved, and the project build scripts have been [updated to no longer
+require a bash analogue][64].)
```

Comment:
I think everything after "installed." can be deleted, along with refs 61 and 64.  We don't really need to point out the history of how we got here.